integration-tests/test/rnode.py

```diff
@@ -290,11 +293,15 @@ def get_mvdag(self) -> str:
     def get_parsed_mvdag(self) -> Dict[str, Set[str]]:
         return parse_mvdag_str(self.get_mvdag())
 
-    def deploy_string(self, rholang_code: str) -> str:
+    def deploy_string(self, rholang_code: str, private_key: Optional[str] = None) -> str:
```

Comment:
After some discussions that I had with @adaszko it seems that this method needs polishing and changes.
Please introduce a `private_key` parameter (as you did) but make it an explicit `String` that everyone has to pass when calling `deploy_string` function. This will require changes in files where `deploy_string` is being used, but the change should be trivial